core-plugins/docs/Deduplicate-batchaggregator.md

```diff
@@ -27,8 +50,9 @@ record will be considered as a whole for deduplication. For example, if the inpu
 that needs to be deduplicated. This property takes in a field name and the logical operation that needs to be performed
 on that field on the set of records. The syntax is `field:function`. For example, if we want to choose the record with
 maximum cost for the records with schema `fname`, `lname`, `item`, `cost`, then this field should be set as `cost:max`.
-Supported functions are `first`, `last`, `max`, and `min`. Note that only one pair of field and function is allowed.
-If this property is not set, one random record will be chosen from the group of 'duplicate' records.
+Supported functions are `any`, `max` and `min`. `min` and `max` must be operated only on numeric filter fields.
+The `first` and `last` are deprecated and will be removed in later versions of the platform. Note that only one pair of 
+field and function is allowed. If this property is not set, one random record will be chosen from the group of 'duplicate' records.
 
 **numPartitions:** An optional number of partitions to use when grouping unique fields. If not specified, the execution
```

Comment:
Can we change the field names to match the UI while we are editing the markdown?  numPartitions can read "Number of Partitions"